src/webapp/components/survey-list/hook/useSurveyListActions.ts

```diff
@@ -182,21 +177,6 @@ export function useSurveyListActions(surveyFormType: SURVEY_FORM_TYPES) {
             changeCurrentCaseReportForm({ id: survey.id, name: survey.name });
     };
 
-    const handleSplitButtonClick = (
-        option:
-            | (typeof PREVALENCE_PATIENT_OPTIONS)[0]
-            | (typeof PREVALENCE_PATIENT_OPTIONS)[1]
-            | (typeof PREVALENCE_PATIENT_OPTIONS)[2]
-            | (typeof PREVALENCE_PATIENT_OPTIONS)[3]
-            | (typeof PREVALENCE_PATIENT_OPTIONS)[4]
```

Comment:
[general comment] This code is being removed but we still have typeof of our code. `typeof` is a code smell when using TS. The only understandable usage of it is when we are dealing with external data, but not with types that we have the power to control. For example: `): FutureData<number | ProgramOptionCountMap>` returned by a use case. Typical solution, ADT: `{type: "value", value: number} | {type: "map: map: ProgramOptionCountMap}`